{{cookiecutter.folder_name}}/Makefile

```diff
@@ -36,12 +37,17 @@ tabdir=./tables
 tables=$(tabdir)/*.tex
 styles=./texmf/styles/*.{sty,cls}
 bibs=./texmf/bib/*.bst
-source=$(main).{tex,bbl,bib} lsstdesc.bib macros.tex authors.tex acknowledgments.tex
+source=$(main).{tex,bbl,bib} lsstdesc.bib authors.tex acknowledgments.tex
```

Comment:
For a different issue, but should `lsstdesc.bib` live in `texmf/bib` (or somewhere else) and be updateable?